### PR TITLE
Postponed/ppna 2579 ppna 2623

### PR DESCRIPTION
**Pull request by TomaszUrugOlszewski**
Public documentation for:
- Goals API
- Custom Dimensions API

Do not merge until **release/99**

### CODE REVIEW

**Comment by RomanSek** [comment]
@TomaszUrugOlszewski I changed destination to 14.0, hope it's ok. You can merge it when you're ready. I looked at API and it looked ok, but you can confirm it here: https://developers.piwik.pro/en/postponed-ppna-2579-ppna-2623/custom_reports/object_management_api/object_management_api.html

**Comment by TomaszUrugOlszewski** [comment]
> @TomaszUrugOlszewski I changed destination to 14.0, hope it's ok. You can merge it when you're ready. I looked at API and it looked ok, but you can confirm it here: https://developers.piwik.pro/en/postponed-ppna-2579-ppna-2623/custom_reports/object_management_api/object_management_api.html

@RomanSek @kedod correct me if im wrong, but it can't be merged to 14.0. It will be available at-some-point-in-the-future, but not in 14.0, or 15.0

**Comment by TomaszUrugOlszewski** [comment]
DO NOT MERGE

File: custom_reports/object_management_api/index.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+swagger: '2.0'
+info:
+  title: Object managemet HTTP API for Analytics (new).
```
Comment (jzyinq) [inline custom_reports/object_management_api/index.yaml:3]: typo `management`

**Comment by TomaszUrugOlszewski** [comment]
yes

**Pull request merged**